include/wx/webview.h

```diff
@@ -220,9 +234,17 @@ class WXDLLIMPEXP_WEBVIEW wxWebView : public wxControl
     //Find function
     virtual long Find(const wxString& text, int flags = wxWEBVIEW_FIND_DEFAULT) = 0;
 
+    //Establish EmulationLevel for RunScript IE
```

Comment:
The first two paragraphs are done. About the latest one, I would do just:
-Default mode, old one, without change registry.
-The eldest mode that works for JSON. Why? My reason is if you use it on WinXP, you are not going to have IE9 and so on. So I will move to 7, that I think is the version that works for this.